### PR TITLE
Add food

### LINKED ISSUE #207: Add food

### PR DESCRIPTION
**Pull request by JesusGonzalezA**
Closes #207
Add new screen to add a new meal
Captura de pantalla 2021-11-15 a las 13 34 34
Captura de pantalla 2021-11-15 a las 13 34 59
Captura de pantalla 2021-11-15 a las 13 35 08
Captura de pantalla 2021-11-15 a las 13 35 15

### CODE REVIEW

**Comment by JesusGonzalezA** [comment]
We can reuse this component in the future to edit already existing meals, it will be just a refactor task.

File: frontend/wasted-app/package.json [hunk 1]
```diff
@@ -9,6 +9,8 @@
   },
   "dependencies": {
     "@expo/vector-icons": "^12.0.5",
+    "@react-native-community/datetimepicker": "^3.5.2",
+    "@react-native-picker/picker": "^2.2.0",
```
Comment (audrius-savickas) [inline frontend/wasted-app/package.json:13]: I think Wix's RN UI Lib already had datetimepicker and picker components, no?

File: frontend/wasted-app/src/api/food.ts [hunk 2]
```diff
@@ -27,3 +27,13 @@ export const getAllFood = async (sortObject?: FoodSortObject): Promise<Food[]> =
...
 }
+
+export const addNewFood = async (food: Food) => {
+  const data = await fetch(`${WASTED_SERVER_URL}/Food`, {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json"
+    },
+    body: JSON.stringify(food)
+  })
```
Comment (audrius-savickas) [inline frontend/wasted-app/src/api/food.ts:38]: I think this function should return null or some error if fetching endpoint returns any error code

File: frontend/wasted-app/package.json (hunk 1 above)
Comment (JesusGonzalezA) [reply frontend/wasted-app/package.json:13]: Yes, I'm using Wix library. It uses native libraries so I need to import those dependencies as the documentation explains
Captura de pantalla 2021-11-15 a las 15 08 17

Comment (audrius-savickas) [reply frontend/wasted-app/package.json:13]: ah gotcha, all good

File: frontend/wasted-app/src/api/food.ts (hunk 2 above)
Comment (JesusGonzalezA) [reply frontend/wasted-app/src/api/food.ts]: Done! We could use some alert library to make this smoothier for the users. For example: https://github.com/acaziasoftcom/react-native-sweet-alert

**Pull request merged**